arraySort.js

```diff
@@ -0,0 +1,22 @@
+function arraySort(a, b) {
+    return b - a
+    //return a<b?1 : a>b?-1 : 0
+}
+
+var x = [1, 4, 7, "x", 3, 5, 8, "t", 0, 6, 10, "a"];
+x.sort(arraySort)
+x.reverse()
+i = 0
```

Comment:
We talked about this previously, Always avoid using variables before declaring them. This is a bad practice in JS it adds them to the global scope